src/app.js

```diff
@@ -2,11 +2,87 @@
  * @param {Object} position
  * @param {Number} position.x
  * @param {Number} position.y
- * 
+ *
  * @returns {Array<position>}
  */
-export function getNearestShops(position) {
-  // code
-  
-  return [];
+import axios from "axios";
+
+const api = "https://blue-bottle-api-test.herokuapp.com/v1"
+const apiHeaders = {'Accept': 'application/json'}
+
+export async function getNearestShops(position) {
+    try {
+        const token = await getTokens()
+        const coffeeShops = await getCoffeeShops(token)
+        const nearShops = await getNearbyShops(position, coffeeShops, 3)
+        console.log('NearbyShops: ', nearShops) // I left this for viewing purposes
```

Comment:
This comment makes it seem like the viewing is just an afterthought. Viewing the results is an important feature, I would prefer to read the output easily.
We did provide an example:

__Expected output__

```
Starbucks Seattle2, 0.0645
Starbucks Seattle, 0.0861
Starbucks SF, 10.0793
```